.github/workflows/publish-swift-package.yaml

```diff
@@ -1,24 +1,66 @@
-name: Build, tag, and release the BreezSDK Swift Package
+name: Publish Swift Package
 on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'Release of the breez-sdk repo to publish (MAJOR.MINOR.PATCH)'
+        description: 'breez-sdk repo release (MAJOR.MINOR.PATCH)'
         required: true
         type: string
 
 jobs:
   build-tag-release:
     name: Build, tag, and release the BreezSDK Swift Package
-    runs-on: macos-12
+    runs-on: macOS-latest
     steps:
+      - name: Install required dependencies
+        run: |
+          brew install protobuf
+          cargo install --version 0.22.0 uniffi_bindgen
```

Comment:
@cnixbtc  Perhaps it worth adding a make target for the dependencies on the sdk-bindings repo so we won't need to maintain the version here as well.
For example now we have this PR: https://github.com/breez/breez-sdk/pull/178 that changes the uniffi version if we remove the specific version here the change would be transparent to this build.

Reply:
I have made some research here and it turns out that uniffi has abandoned the uniffi-bindgen as a library and there are other instructions to build it from the uniffi-crate ([nee](https://mozilla.github.io/uniffi-rs/tutorial/foreign_language_bindings.html#creating-the-bindgen-binary))

I will handle this on this PR and after I will remove that dependency we can remove it here as well but I think it is not part of this PR to you can ignore the above.

Reply:
Yeah that makes sense!

Just let me know once the dependency can be removed, I'll update the build script then in a new PR. 👍